src/prefect/cli/_prompts.py

```diff
@@ -238,6 +242,21 @@ def process_response(self, value: str) -> str:
             raise InvalidResponse(self.validate_error_message)
 
 
+def prompt_for_max_active_runs_and_catchup(console):
+    """
+    Prompt the user for the maximum number of active runs and whether to catch up on missed runs.
+    """
+    max_active_runs = prompt(
+        "Maximum number of active runs (leave blank for unlimited)",
+        default=None,
+    )
+    catchup = confirm(
+        "Catch up on missed runs?",
```

Comment:
Maybe include the fact that we're talking about 'late' runs here?

```suggestion
        "Catch up on late flow runs?",
```